assets/component-slider.css

```diff
@@ -32,6 +32,24 @@ slider-component {
   }
 }
 
+@media screen and (min-width: 750px) {
+  .slider.slider--tablet-up {
+    position: relative;
+    flex-wrap: inherit;
+    overflow-x: auto;
+    scroll-snap-type: x mandatory;
+    scroll-behavior: smooth;
+    scroll-padding-left: 1rem;
+    -webkit-overflow-scrolling: touch;
+    margin-bottom: 1rem;
```

Comment:
Due to this margin-bottom, it looks like the slider buttons aren't quite vertically aligned: [video](https://screenshot.click/21-12-wnay2-9vv9z.mp4)